cmake/Config.cmake

```diff
@@ -81,7 +81,7 @@ endif()
 # detect the compiler and its version
 # Note: on some platforms (OS X), CMAKE_COMPILER_IS_GNUCXX is true
 # even when CLANG is used, therefore the Clang test is done first
-if(CMAKE_CXX_COMPILER MATCHES ".*clang[+][+]" OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+if(CMAKE_CXX_COMPILER MATCHES ".*clang[+][+]" OR CMAKE_CXX_COMPILER_ID MATCHES ".*Clang")
```

Comment:
The `.*` isn't really needed here (and will in theory slow down the expression), because `MATCHES` will just match as a sub string just fine, if you're not using `^` or `$`. So looking for `Clang` will match both `AppleClang`  as well as `Clang` or `Clangtastic`.

For the same reasons you could actually use this opportunity to also simplify the first expression, only looking for `clang`.

Reply:
Indeed, thanks!